src/recovery/delivery_rate.rs

```diff
@@ -247,7 +255,9 @@ mod tests {
             is_app_limited: false,
         };
 
-        recvry.delivery_rate.on_packet_sent(&mut pkt_1, Instant::now());
+        recvry
```

Comment:
nit: typo in `recovry`

Reply:
:man_shrugging:  that's how the variable is called in the pre-existing test...